packages/mobile/src/app/reducers.ts

```diff
@@ -57,6 +57,7 @@ export interface State {
   showPriceChangeIndicatorInBalances: boolean
   paymentDeepLinkHandler: PaymentDeepLinkHandler
   dappsWebViewEnabled: boolean
+  dappSessionActive: Dapp | null
```

Comment:
i think this is a good idea, but i didn't do this because we don't store the list of dapps in redux so having the whole dapp here didn't feel like data duplication. the only dapps data we store is in the recent dapps list but it seems unsafe to tie this feature to the recent dapps (the number of recent dapps is controlled by remote config and could be 0, in which case we wouldn't store any dapps)

Reply:
ah yeah right! all good then 👍